docs/deploy/installation.md

```diff
@@ -43,9 +43,134 @@ Instead of depending on IMDSv2, you can specify the AWS Region via the controlle
 
 The controller runs on the worker nodes, so it needs access to the AWS ALB/NLB APIs with IAM permissions.
 
-The IAM permissions can either be setup using [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) or can be attached directly to the worker node IAM roles. The best practice is using IRSA if you're using Amazon EKS. If you're using kOps or self-hosted Kubernetes, you must manually attach polices to node instances.
+The IAM permissions can be setup using any of:
 
-### Option A: Recommended, IAM roles for service accounts (IRSA)
+- [Pod Identity](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html) associations
+- [IAM roles for service accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
+- Attached IAM permissions directly to the worker node IAM roles. 
+
+The best practice is using Pod Identity associations if you're using Amazon EKS. If you're using kOps or self-hosted Kubernetes, you must manually attach polices to node instances.
+
+### Required IAM policy
+
+The IAM policy that must be attached to a service account may differ based on the region. Download an IAM policy for AWS Load Balancer Controller for your region using one of the following commands:<p>
+    If your cluster is in a US Gov Cloud region:
+    ```
+    curl -o iam-policy.json https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.9.0/docs/install/iam_policy_us-gov.json
```

Comment:
nit: change to the latest version's [url](https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/main/docs/install/iam_policy.json), and potentially show an example of specific version if someone needs to.